frontend/src/store/sc/reducer.ts

```diff
@@ -0,0 +1,29 @@
+import { DataStatus } from 'common/enums/enums';
```

Comment:
reducer & actions should be by page. move it to the `frontend/src/store/sc-configurate-instance/actions.ts`